src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs

```diff
@@ -39,14 +39,14 @@ private async Task<(TestChainContext context, IPEndPoint peerEndPoint)> Initiali
             return (context, peerEndPoint);
         }
 
-        [Fact]
+        [Fact(Skip = "Revisit with ConsensusManager tests")]
```

Comment:
Hey are these tests happening for ConsensusManager? Can delete from here now?

Reply:
Well we still did not write the unit tests for CM.